templates/docs/examples/layouts/application/structure.html

```diff
@@ -13,13 +13,14 @@
     <div class="u-clearfix"><code>l-navigation-bar</code> <button class="u-float-right js-menu-toggle is-dense u-no-margin">Menu</button></div>
   </div>
 
-  <header class="l-navigation">
+  <header class="l-navigation is-dark">
```

Comment:
This shouldn't be needed, right?

Reply:
No it shouldn't, overlooked it - removed!